uint8 raw_data    # The raw value of the EAG DAC, in the range of 0-255.
uint64 timestamp  # The absolute time at which this measurement was received by the pixhawk.
